### PR TITLE
Document new permissions

### PR DESCRIPTION
**Pull request by matklad**
bors r+
🤖

### CODE REVIEW

File: docs/dev/README.md [hunk 1]
```diff
@@ -292,3 +292,17 @@ Release steps:
...
+There are three sets of people with extra permissions:
+
+* rust-analyzer GitHub organization **admins** (which include current t-compiler leads).
+  Admins have full access to the org.
+* **review** team in the organization.
+  Reviewers have `r+` access to all of organization's repositories and publish rights on crates.io.
+  They also have direct commit access, but all changes should via bors queue.
+  It's ok to self-approve if you think you know what you are doing!
+  bors should automatically sync the permissions.
+* **triage** tam in the organization.
```
Comment (Veykril) [inline docs/dev/README.md:307]:
```suggestion
* **triage** team in the organization.
```

File: docs/dev/README.md [hunk 2]
```diff
@@ -292,3 +292,17 @@ Release steps:
...
+
+# Permissions
+
+There are three sets of people with extra permissions:
+
+* rust-analyzer GitHub organization **admins** (which include current t-compiler leads).
+  Admins have full access to the org.
+* **review** team in the organization.
+  Reviewers have `r+` access to all of organization's repositories and publish rights on crates.io.
+  They also have direct commit access, but all changes should via bors queue.
```
Comment (bjorn3) [inline docs/dev/README.md:304]: should **go** via **the** bors

**Pull request merged**